shotover-proxy/src/transforms/protect/crypto.rs

```diff
@@ -42,7 +42,7 @@ pub async fn encrypt(
     // TODO: investigate using storing as blob instead of text and then use textAsBlob here
     // We should functionally verify which has better performance before sticking with one.
     Ok(Operand::Const(format!(
-        "'{}'",
+        "textAsBlob('{}')",
         serde_json::to_string(&protected)?
```

Comment:
To actually make use of storing as a blob we need to encode with bincode or similar instead of json.

Reply:
Isn't this storing a JSON string as blob within Cassandra?

Reply:
correct.
Storing json as a blob or a text should take up the same amount of space.

Reply:
Wouldn't it going from bincode -> string -> bincode then?

Reply:
Oh I see where it went wrong.
We shouldnt be using textAsBlob.
Instead we can insert directly to the blob column by writing our binary data as a hex string.

https://stackoverflow.com/questions/22141620/is-it-possible-in-cassandra-cql3-insert-statement-binary-data

Reply:
the struct wouldn't be valid hex.

Reply:
the struct?
The struct gets encoded into a Vec<u8> by bincode.
Then we turn the Vec<u8> into a hex encoded String